api-docs/index.html

```diff
@@ -149,85 +153,76 @@
                         <ul id="nav-joomla" class="nav">
                             <li class="dropdown">
                                 <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-                                    <span dir="ltr"><span aria-hidden="true" class="icon-joomla"></span> Joomla!<sup>&reg;</sup></span> <span class="caret"></span>
+                                    <span dir="ltr"><span aria-hidden="true" class="icon-joomla"></span> Joomla!<sup>®</sup></span> <span class="caret"></span>
```

Comment:
it seems you copied the browser version and not the source code, at least that's how it looks compared to joomla.org

Reply:
ok i will make the change

Reply:
else everything is fine or any more changes are needed?